requirements.txt

```diff
@@ -1,2 +1,2 @@
 jinja2>=3.1.2,<3.2
-zimscraperlib>=1.8.0,<1.9
+zimscraperlib>=3.0.0
```

Comment:
I have changed the requirements.txt to require scraperlib version 3.0.0 .

Reply:
Look at the line you changed ; it was a range. We want to use a range as well.

Reply:
No. What's the purpose of the change? To allow updates but only up to a certain point so we don't get breaking changes

Reply:
So, it should be >2.1.0,<=3.0.0 as the long_description hasnt been added in 2.1.0 which was the previous version and we need 3.0.0 as it is the first version on scraperlib that supports long_descrition

Reply:
Great, I'll change it.

Reply:
I changed it in the last commit.